test/complex.jl

```diff
@@ -73,9 +73,9 @@
 
     @test subs(4x + 8y^2 - 6x^3, [x, y] => [2 + 4im, 9 - im]) ==
           (1176 - 32im) * x^0
-    @test subs(4x + 8y^2 - 6x^3, [x, conj(y)] => [2 + 4im, 9 - im]) ==
+    @test_broken subs(4x + 8y^2 - 6x^3, [x, conj(y)] => [2 + 4im, 9 - im]) ==
```

Comment:
Add a comment explaining where this is coming. You also need to fix the formatting